example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -610,7 +638,7 @@ describe('The S3 Ingest Granules workflow', () => {
           // protect against "undefined": TaskStateEntered has "input" but not "name"
           if (event.name && intersection(eventKeys, ['input', 'output']).length === 1) {
             // each step should contain status information
-            if (event.type === 'TaskStateExited') {
+            if (event.type === 'TaskStateEited') {
```

Comment:
is this a typo?

Reply:
yeah it must be because i didn't change that on purpose